tests/trainer/test_trainer.py

```diff
@@ -791,6 +792,8 @@ def test_tf32(self):
 @require_sentencepiece
 @require_tokenizers
 class TrainerIntegrationTest(TestCasePlus, TrainerIntegrationCommon):
+    FLAG_ACCELERATOR_SUPPORT_GRAD_ACCUM_KWARGS = is_accelerate_available("0.28")
```

Comment:
```suggestion
    GRAD_ACCUM_KWARGS_VERSION = is_accelerate_available("0.28")
```

Let's be a bit more explicit in what this actually is

Reply:
changed it to what you suggested, but would `GRAD_ACCUM_KWARGS_VERSION_AVAILABLE`  be even more explicit?